docker/packaging/package_vitess.sh

```diff
@@ -1,61 +0,0 @@
-#!/bin/bash
```

Comment:
What are you replacing this with ?

Reply:
I don't believe this is used (the clue to me was specific about the binaries it packages). The source to Gary's scripts is here: https://github.com/planetscale/vitess-releases/blob/master/bin/

Reply:
@gedgar confirmed this file is not used.

Reply:
:+1: